### PR TITLE
Home and playlist UI

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this PR, 
UI screens for the home page and playlist page have been added. 
Below are the screenshots for both pages.
Screenshot (136)
Screenshot (137)
There is some code which is commented. That I will remove later.
Please review and suggest improvements.

### CODE REVIEW

**Review by Omkar-Chaskar — APPROVED** [review]
UI looks good and code is also clean.
Just a small suggestion , 
you can use React icons instead of images.
It will make easier to load and render icons on webpage rather than image which size is more than 10kb.

File: src/pages/history/history.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { HistoryCard } from '../../components/historyCard/historyCard';
+import { Navbar } from '../../components/navbar/navbar';
```
Comment (Manju-sree) [inline src/pages/history/history.jsx:2]: you can create index.js file for all components to import @once like import { HistoryCard,Navbar } from '../../components/index'

File: src/pages/home/home.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  ::-webkit-scrollbar-track {
+    background: #f1f1f1; 
+  }  
+
+  ::-webkit-scrollbar-thumb {
+    background: var(--secondary-bg-dark-color); 
+  }
+
+/* .app-container {
```
Comment (Manju-sree) [inline src/pages/home/home.css:16]: remove commented code if not necessary

**Review by Manju-sree — APPROVED** [review]
please check few suggestions are commented below, apart from that this PR looks fine to merge.

**Comment by ShreyashMahajan** [comment]
> UI looks good and code is also clean. Just a small suggestion , you can use React icons instead of images. It will make easier to load and render icons on webpage rather than image which size is more than 10kb.

thank you, i will do that

File: src/pages/home/home.css (hunk 2 above)
Comment (ShreyashMahajan) [reply src/pages/home/home.css:16]: ok

File: src/pages/history/history.jsx (hunk 1 above)
Comment (ShreyashMahajan) [reply src/pages/history/history.jsx:2]: thank you, i will implement that

**Pull request merged**